### PR TITLE
Fix most failures in doctest

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Fixes most doctest failures. This is a temporary solution -- yesterday I got a mock DWaveSampler working inside local doctests, more or less, so expect a next PR to expand the coverage of doctests that is reduced by this PR.

### CODE REVIEW

File: docs/examples/and.rst [hunk 1]
```diff
@@ -220,13 +220,13 @@ the problem. Its last line prints a confirmation that indeed the two selected qu
...
 >>> from dwave.system.composites import FixedEmbeddingComposite
->>> sampler_embedded = FixedEmbeddingComposite(sampler, {'x': [0], 'z': [4]})
+>>> sampler_embedded = FixedEmbeddingComposite(sampler, {'x': [0], 'z': [4]})   # doctest: +SKIP
 >>> print(sampler_embedded.adjacency)     # doctest: +SKIP
 {'x': {'z'}, 'z': {'x'}}
 
 As before, we ask for 5000 samples.
 
->>> response = sampler_embedded.sample_qubo(Q_not, num_reads=5000)
+>>> response = sampler_embedded.sample_qubo(Q_not, num_reads=5000)      # doctest: +SKIP
```
Comment (arcondello) [inline docs/examples/and.rst]: I think we should start the process of switching these over to `sampleset`

File: docs/examples/and.rst [hunk 2]
```diff
@@ -220,13 +220,13 @@ the problem. Its last line prints a confirmation that indeed the two selected qu
...
 >>> for datum in response.data(['sample', 'energy', 'num_occurrences']):   # doctest: +SKIP
```
Comment (arcondello) [inline docs/examples/and.rst]: Also think we should start using the `print(sampletset)` output

File: docs/examples/and.rst (hunk 1 above)
Comment (JoelPasvolsky) [reply docs/examples/and.rst]: I agree but not in this PR. I'll be going through all examples as part of my next PR to use a mock sampler in doctests and that will be the right time to update the examples, as I'll be doing that anyway to extend the doctest coverage by adjusting outputs to reliably reproducible ones

File: docs/examples/and.rst (hunk 2 above)
Comment (JoelPasvolsky) [reply docs/examples/and.rst]: Yes

**Comment by JoelPasvolsky** [comment]
Submodules still to be updated: 
* https://github.com/dwavesystems/dwave_networkx/pull/161
* https://github.com/dwavesystems/dwave-neal/pull/76
* https://github.com/dwavesystems/dwave-cloud-client/pull/372
* https://github.com/dwavesystems/dwave-hybrid/pull/222
* https://github.com/dwavesystems/dwavebinarycsp/pull/100  -- MERGED

**Pull request merged**